src/command/cmd_defs.c

```diff
@@ -2545,7 +2545,7 @@ static const struct cmd_t command_defs[] = {
               "/executable urlsave set \"curl %u -o %p\"",
               "/executable urlsave default",
               "/executable vcard_photo set \"feh %p\"",
-              "/executable editor set vim")
+              "/executable editor set \"emacsclient -t\"")
```

Comment:
```suggestion
              "/executable editor set <editor>",
              "/executable editor set \"<editor> -args\"")
```

let's maybe change this to a less opinionated/more generic example?

Reply:
sounds good.
I don't know if it will work an editor without being inside double quotes (such as `/executable editor set vim` instead of `/executable editor set "vim"`).

Reply:
I actually think as an example this is fine. I think all our (most?) examples are practical. See feh, firefox etc. We could add a generic part into `CMD_SYN` though.

Reply:
I actually do believe the `emacsclient -t` is what some emacs user will want. So I think it's a good example.
Sure just to show how to pass a flag via "" would be enough. But I don't see any harm in pointing them to the thing that they might actually want to be doing anyways.

But actually (not related to your changes) the `/executables editor` is missing anyways from `CMD_SYN`. Would you mind to add the generic way there?